sdks/java/core/src/main/java/org/apache/beam/sdk/util/construction/Environments.java

```diff
@@ -488,6 +488,7 @@ public static Set<String> getJavaCapabilities() {
     capabilities.add(BeamUrns.getUrn(Primitives.TO_STRING));
     capabilities.add(BeamUrns.getUrn(StandardProtocols.Enum.DATA_SAMPLING));
     capabilities.add(BeamUrns.getUrn(StandardProtocols.Enum.SDK_CONSUMING_RECEIVED_DATA));
+    capabilities.add(BeamUrns.getUrn(StandardProtocols.Enum.ORDERED_LIST_STATE));
```

Comment:
FWIW, adding this to the capabilities list should be the *last* thing we do, once we've verified everything else works end-to-end.

Reply:
Removing the validation checks on the backend, I can run a JAVA pipeline with OrderedListState. So it seems the plumbing is all in place.